lrpar/src/lib/ctbuilder.rs

```diff
@@ -1322,9 +1321,24 @@ where
                 }
             }
 
-            outs.push_str("\n    }\n\n");
+            let action_args = str::parse::<TokenStream>(&outs)?;
```

Comment:
I think I'd rename this `action_body`, while it is parsing the `$args` it is really doing so within the body.

Reply:
Fixed in [bc995e0](https://github.com/softdevteam/grmtools/pull/519/commits/bc995e0e415f5f450342424b819d8431c5886319)